src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller.cs

```diff
@@ -152,7 +152,7 @@ internal override void OnModelUpdated(Model modelOpt)
 
         private bool StartNewModelComputation(
             CompletionService completionService,
-            CompletionTrigger trigger)
+            CodeAnalysis.Completion.CompletionTrigger trigger)
```

Comment:
:bulb: Prefer a using alias declaration to qualification at the use site (applies to other cases as well)

Reply:
fixed